packages/augur-ui/src/modules/reporting/common.tsx

```diff
@@ -463,7 +471,8 @@ export class ReportingBondsView extends Component<
       reportingGasFee,
       reportAction,
       preFilledStake,
-      updatePreFilledStake
+      updatePreFilledStake,
+      rep
```

Comment:
If this is userRep, just a suggestion change to ‘userAvailableRep or if its already formatted change to userRepFormatted.